<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="xLabel1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>x():</string>
   </property>
  </widget>
  <widget class="QLabel" name="yLabel1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>y():</string>
   </property>
  </widget>
  <widget class="QLabel" name="frameGeometry1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>frameGeometry():</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>pos():</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>geometry():</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>width():</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>height():</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>rect():</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>size():</string>
   </property>
  </widget>
  <widget class="QLabel" name="xLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="yLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="frameGeometry">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>140</width>
     <height>12</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="pos">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>140</width>
     <height>12</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="geometry">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>140</width>
     <height>12</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="width">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>140</width>
     <height>12</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="height">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="rect">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>140</width>
     <height>12</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="size">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>140</width>
     <height>12</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
